src/analysis/special_functions/pow.lean

```diff
@@ -1583,6 +1583,11 @@ begin
     exact ennreal.continuous_inv.continuous_at.comp (continuous_at_rpow_const_of_pos z_pos) }
 end
 
+protected lemma tendsto.rpow {α : Type*} {f : filter α} {m : α → ℝ≥0∞} {a : ℝ≥0∞} (r : ℝ)
```

Comment:
I see we already have:

- `filter.tendsto.cpow`
- `filter.tendsto.rpow`
- `filter.tendsto.nnrpow`
- `filter.tendsto.const_cpow`
- `filter.tendsto.rpow_const`

Based on this I think we should:

- Rename `filter.tendsto.const_cpow` to `filter.tendsto.cpow_const`
- Call this lemma `filter.tendsto.ennrpow_const` rather than the proposed `ennreal.tendsto.rpow`
- Add an appropriate lemma named `filter.tendsto.ennrpow`

Reply:
I don't quite understand why we should change `filter.tendsto.const_cpow` to `filter.tendsto.cpow_const` since the lemma is about taking a constant to the power of a function

Reply:
Oops you're right; we definitely should not be renaming `filter.tendsto.const_cpow`.

However it still seems like `filter.tendsto.ennrpow_const` would be a more consistent name for the new lemma, even bearing in mind that it mixes types by raising an ennreal-valued function to a real.

Reply:
I will change the name to what you suggested. I tried to prove `filter.tendsto.ennrpow` but it seems that we are missing a few continuity lemmas. That should probably belong in another PR.

Reply:
Yep, it can wait!